calour/experiment.py

```diff
@@ -53,10 +53,12 @@ class Experiment:
         The metadata on the features
     exp_metadata : dict
         metadata about the experiment (data md5, filenames, etc.)
-    shape: tuple of (int, int)
+    shape : tuple of (int, int)
         the dimension of data
+    sparse : bool
+        store the data as sparse or dense array.
     description : str
-        name of the experiment
+        description of the experiment
```

Comment:
i think name is more appropriate, as it is supposed to be a short name of the experiment
(to enable identification if you have many exps...)?